jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannel.java

```diff
@@ -298,8 +299,73 @@ public EndPoint getEndPoint()
         return _endPoint;
     }
 
+    /**
+     * <p>Return the Local Name of the connected channel.</p>
+     *
+     * <p>
+     * This is the name after the connector is bound and the connection is accepted.
+     * The value is typically the local address and optionally the local host name
+     * if the local address is unavailable.
+     * </p>
+     * <p>
+     * Value can be overridden by {@link HttpConfiguration#setLocalAddress(SocketAddress)} for
+     * scenarios where Jetty is being an intermediary and the local name
+     * needs to be changed to satisfy public (pre-intermediary) HTTP behaviors
+     * such as absolute-URI creation (eg: Location response header).
+     * </p>
```

Comment:
This paragraph is incorrect. It's not used for URI creation (and should not be).